techniques/general/G95.html

```diff
@@ -12,119 +11,98 @@
+            <li><a href="#description">解説</a></li>
+            <li><a href="#examples">事例</a></li>
+            <li><a href="#related">関連する達成方法</a></li>
+            <li><a href="#tests">検証</a></li>
          </ul>
       </nav>
-      <h1>Providing short text alternatives that provide a brief description of
-         the non-text content
+      <h1>非テキストコンテンツの簡単な説明を提供する、簡潔なテキストによる代替を提供する
       </h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://w3c.github.io/wcag/understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques.html">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Applies to all technologies</p>
-            <p>This technique relates to <span><a href="https://w3c.github.io/wcag/understanding/non-text-content">Success Criterion 1.1.1: Non-text Content</a> (Sufficient)</span>.
+            <h2>適用 (対象)</h2>
+            <p>全てのウェブコンテンツ技術</p>
+            <p>これは <span><a href="https://w3c.github.io/wcag/understanding/non-text-content">達成基準 1.1.1: 非テキストコンテンツ</a> (十分な達成方法)</span> に関する達成方法である。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>This technique is used when the text needed to serve the same purpose and
-               present the same information as the original non-text content is too lengthy
-               or when this goal cannot be achieved with text alone. In that case this
-               technique is used to provide a short text alternative that briefly describes
-               the non-text content. (A long text alternative is then provided using
-               another technique such that the combination serves the same purpose and
-               presents the same information as the original non-text content.)
+            <h2>解説</h2>
+            <p>この達成方法の目的は、元の非テキストコンテンツと同じ目的を果たし、同じ情報を提示しようとするとテキストが長くなりすぎてしまう場合、又はテキストのみではこの目的を達成できない場合に用いられる。そのような場合に、非テキストコンテンツを簡単に説明する簡潔なテキストによる代替を提供するために、この達成方法を用いる。(長いテキストによる代替は、非テキストコンテンツと同じ目的を果たし、同じ情報を提示するという別の達成方法を併用することで提供される。)
```

Comment:
```suggestion
            <p>この達成方法の目的は、元の非テキストコンテンツと同じ目的を果たし、同じ情報を提示しようとするとテキストが長くなりすぎてしまう場合、又はテキストのみではこの目的を達成できない場合に用いられる。そのような場合に、非テキストコンテンツを簡単に説明する簡潔なテキストによる代替を提供するために、この達成方法を用いる。(その後、長いテキストによる代替は、元の非テキストコンテンツと同じ目的を果たし、かつ同じ情報を提示するような別の達成方法を用いて提供される。)
```

Reply:
```suggestion
            <p>この達成方法の目的は、元の非テキストコンテンツと同じ目的を果たし、同じ情報を提示しようとするとテキストが長くなりすぎてしまう場合、又はテキストのみではこの目的を達成できない場合に用いられる。そのような場合に、非テキストコンテンツを簡単に説明する簡潔なテキストによる代替を提供するために、この達成方法を用いる。(長いテキストによる代替は、非テキストコンテンツと同じ目的を果たし、同じ情報を提示するという別の達成方法を併用することで提供される。)
```

Reply:
```suggestion
            <p>この達成方法は、元の非テキストコンテンツと同じ目的を果たし、同じ情報を提示しようとするとテキストが長くなりすぎてしまう場合、又はテキストのみではこの目的を達成できない場合に用いられる。そのような場合に、非テキストコンテンツを簡単に説明する簡潔なテキストによる代替を提供するために、この達成方法を用いる。(その後、長いテキストによる代替は、元の非テキストコンテンツと同じ目的を果たし、かつ同じ情報を提示するような別の達成方法を用いて提供される。)
```

Reply:
@caztcha 語順を触ってみました。問題なければマージしてもらえればと思います。

```suggestion
            <p>この達成方法は、元の非テキストコンテンツと同じ目的を果たし、同じ情報を提示するために必要なテキストが長くなりすぎてしまう場合、又はテキストのみではこの目的を達成できない場合に用いられる。そのような場合、非テキストコンテンツを簡潔に説明する短いテキストによる代替を提供するために、この達成方法は用いられる。(その後、組み合わせによって元の非テキストコンテンツと同じ目的を果たし、かつ同じ情報を提示するような、別の達成方法を用いて、長いテキストによる代替が提供される。)
```